storybookjs/storybook
# addon-knobs radio input

Issue:

## What I did

Added 'radio' knob to addon-knobs. This is functionally the same as the 'select' knob, however the UI is build with `<input type="radio">` so the options are presented in a horizontal list with radio inputs. 

An ideal use-case for this feature is a theme switcher - e.g. light and dark theme, etc. This is a superior user experience than the 'select' knob, as the selection can be more easily changed by single mouse clicks or using the arrow keys. 

## How to test

Is this testable with Jest or Chromatic screenshots?

I have included the [Radio.js test file](https://github.com/astrotim/storybook/blob/addon-knobs-radio-input/addons/knobs/src/components/__tests__/Radio.js) for testing with Jest. 

Does this need a new example in the kitchen sink apps?

No.

Does this need an update to the documentation?

Yes. The README has been updated with relevant docs.

If your answer is yes to any of these, please make sure to include it in your PR.

For maintainers only: Please tag your pull request with at least one of the following:
`["cleanup", "BREAKING CHANGE", "feature request", "bug", "documentation", "maintenance", "dependencies", "other"]`

**Example**

![example](http://astronautweb.co/wp-content/uploads/2018/07/radio-input.gif)
